src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -43,6 +52,9 @@ const SkillsQuizStepper = () => {
                 edX is here to help you find the course(s) or program(s) to help you take the next step in your career.
                 Tell us a bit about your current role, and skills or jobs you&apos;re interested in.
               </p>
+              <GoalDropdown handleGoalOptionChange={handleGoalOptionChange} />
+              { showSearchJobsAndSearchResults ? <SearchJobDropdown /> : null }
```

Comment:
I think we can do it this way as we are showing both components on the basis of same variable.
```
{ 
showSearchJobsAndSearchResults ? 
<SearchJobDropdown />
<SearchResults />
 : null
  }
```